src/acceptance/helpers/debug.go

```diff
@@ -29,6 +29,7 @@ func DebugInfo(cfg *config.Config, setup *workflowhelpers.ReproducibleTestSuiteS
 		commands = append(commands, command("cf", "autoscaling-metrics", anApp, "responsetime"))
 		commands = append(commands, command("cf", "autoscaling-metrics", anApp, "throughput"))
 		commands = append(commands, command("cf", "autoscaling-metrics", anApp, "cpu"))
+		commands = append(commands, command("cf", "autoscaling-metrics", anApp, "cpuutil"))
```

Comment:
unrelated: just a little improvement of the debugging output of the acceptance tests